ghconfig.py

```diff
@@ -69,20 +67,43 @@ def remove_branch_protection(organisation, repository, branch):
     return response.status_code
 
 
+def load_configuration(configuration_file):
+    with open(configuration_file) as config_file:
+        configuration = yaml.load(config_file)
```

Comment:
I totally agree! I did even read it in the documentation and I forgot to change it :O